superset/databases/commands/exceptions.py

```diff
@@ -176,3 +176,8 @@ class DatabaseOfflineError(SupersetErrorException):
 
 class InvalidParametersError(SupersetErrorsException):
     status = 422
+
+
+class SSHTunnelCreateFailedError(CommandException):
```

Comment:
This class should be in `databases/ssh_tunnel/commands/exceptions.py` like the others.